client/src/featureform/enums.py

```diff
@@ -142,13 +142,15 @@ def is_supported(cls, file_path: str) -> bool:
         return False
 
     @classmethod
-    def get_format(cls, file_path: str) -> str:
+    def get_format(cls, file_path: str, default: str = "") -> str:
         file_name = path.basename(file_path)
 
         for file_format in cls:
             if fnmatch(file_name, f"*.{file_format.value}"):
                 return file_format.value
 
+        if default != "":
```

Comment:
Whats this default logic doing?

Reply:
The idea is that `get_format` would fail if you don't provide it a default format; thus, using the empty string. And if you have a default value then it would default to that value if it doesn't pick up a certain format; ex. directories

Reply:
I guess do we even need it if we're just returning the value we give it?

Reply:
I had it separately then Ali wanted it within this class. So moved it here.